pyccel/ast/headers.py

```diff
@@ -696,8 +798,8 @@ def apply(self, args, results=None):
                     new = d_results[arg]
                 else:
                     new = arg
-               #TODO uncomment later
-               #     raise ValueError('Unknown variable name')
+                #TODO uncomment later
+                #     raise ValueError('Unknown variable name')
```

Comment:
Do we still need this? I couldn't work out what it was for

Reply:
Fixed here 9e3988d